src/en/server-hosting/popular-forks-cdns.md

```diff
@@ -0,0 +1,46 @@
+# List of codebases with prebuilt server packages
+
+This document will serve as a list of forks known to have a prebuilt server software for players to download.
+
+This doc was last updated at 2024-10-01
+
+```admonish danger
+All forks listed here (Exept Wizard's Den) are **not vetted by Wizard's Den**. We only just put the links in a convinient place.
```

Comment:
```suggestion
All forks' builds listed here (Except Wizard's Den) are **not vetted by the Space Wizards**. We only put the links here for convenience.
```

Reply:
Maybe it should say "not vetted by Space Wizards **management**" since Space Wizards includes anyone including contributors.